src/security/jwt_manager.py

```
@@ -78,16 +85,16 @@ def create_refresh_token(self, user_data: Dict[str, Any]) -> str:
         }
         return jwt.encode(payload, self.secret_key, algorithm=self.algorithm)
     
-    def create_token_pair(self, user_data: Dict[str, Any]) -> TokenResponse:
-        """Create both access and refresh tokens"""
+    def create_token_pair(self, user_data: Dict[str, Any]) -> TokenPair:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)